core/modules/user/user.admin.inc

```diff
@@ -168,6 +168,48 @@ function user_admin_settings($form, &$form_state) {
     }
   }
 
+  // Password settings.
```

Comment:
I believe these settings should be under Login Settings rather than Account Settings. So they should be here /admin/config/people/login

Reply:
Nope. Password constrains are being applied when a user account is being created - not when the password is being used to log in. It's fine under Account Settings.